### DISCUSSION TITLE
Targeting all resources

### DISCUSSION THREAD

**Discussion by lingrlongr** [cloud-custodian/cloud-custodian#7511] [AWS] [answered]
Is there any way to target all resources with a policy?  My hope was that there was a way to use `value_from` to introduce a list of AWS allowed resource types to a policy, and if a resource exists that isn't in that list, delete it unless it was tagged with a particular key/value.  It seems though that `resource` is a required field and `*` or `all` aren't valid resource types for provider `aws`.

Thanks!

**Comment by thisisshi** [comment, marked as answer]
Hi @lingrlongr there isn't a way to wildcard out the resource type. If you're looking for a way to prevent users from creating resources that are not in a certain list, I'd recommend implementing iam policies that restrict api actions only on certain services or org level SCPs
